z3/src/solver.rs

```diff
@@ -73,6 +73,11 @@ impl<'ctx> Solver<'ctx> {
         }
     }
 
+    /// Return the context of the solver.
+    pub fn get_ctx(&self) -> &'ctx Context {
```

Comment:
```suggestion
    pub fn get_context(&self) -> &'ctx Context {
```

I think it is a little cleaner to avoid contractions / shortening words in public APIs within reason.